go.mod

```diff
@@ -1,11 +1,11 @@
 module github.com/mohammed90/caddy-aws-transport
 
-go 1.20
+go 1.21.10
 
 require (
 	github.com/BishopFox/aws-signing v1.0.1
-	github.com/aws/aws-sdk-go-v2 v1.2.0
-	github.com/caddyserver/caddy/v2 v2.7.0
+	github.com/aws/aws-sdk-go-v2 v1.26.1
+	github.com/caddyserver/caddy/v2 v2.7.6
```

Comment:
Can you keep Caddy at v2.7.0? It'll allow builds of Caddy across all versions from v2.7.0 and higher.

Reply:
Good point. I'll check everything again (just in case) and push an update.

Reply:
@mohammed90 I've replaced the commit with an update that retains the caddy version.